schedule/qam/common/qem-live-migration.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: live_migration
+description:    >
+    scheduling tests of live migration
+schedule:
+   - '{{host_install}}'
+   - '{{migrate_src}}'
+   - '{{migrate_dst}}'
+conditional_schedule:
+   host_install:
+      HOST_INSTALL:
+         1:
+            - autoyast/prepare_profile
+            - boot/boot_from_pxe
+            - autoyast/installation
+            - virt_autotest/login_console
+   migrate_src:
+      MIGRATION_ROLE:
+         src:
+            - virtualization/livemigration/live_migration_src
+   migrate_dst:
+      MIGRATION_ROLE:
+         dst:
+            - virtualization/livemigration/live_migration_dst
+
+test_data:
+   location:
+      "sles-15": "http://dist.suse.de/install/SLP/SLE-15-Installer-LATEST/x86_64/DVD1/"
+      "sles-15-sp1": "http://dist.suse.de/install/SLP/SLE-15-SP1-Installer-LATEST/x86_64/DVD1/"
```

Comment:
Regarding the released SLE version should use GM without LATEST
For example: 
http://dist.suse.de/install/SLP/SLE-15-SP1-Installer-GM/x86_64/

Reply:
will use GM.